roles/docker-cleanup/files/docker-cleanup.service

```diff
@@ -7,6 +7,7 @@ ConditionPathExists=/etc/docker-gc-exclude-containers
 Type=oneshot
 ExecStartPre=-/usr/bin/docker rm -v docker-gc
 ExecStartPre=/bin/echo "Starting disk usage" ; /bin/df -h
+ExecStart=-/usr/bin/bash -c "/usr/bin/docker rm $(/usr/bin/docker ps -qaf 'status=dead')"
```

Comment:
Might not be a bad idea to do `rm -v`, though the `docker-cleanup-volumes` should catch it.